nixos/modules/services/misc/radicle.nix

```diff
@@ -0,0 +1,515 @@
+            };
+            db = {
+              journalMode = mkOption {
+                type = types.str;
+                default = "rollback";
+                description = "";
+              };
+            };
+            externalAddresses = mkOption {
+              type = with types; listOf str;
+              default = [];
+              description = "Specify the node's public addresses";
+            };
+            limits = {
+              connection = {
+                inbound = mkOption {
+                  type = types.ints.unsigned;
+                  default = 128;
+                  description = "Max inbound connections";
+                };
+                outbound = mkOption {
+                  type = types.ints.unsigned;
+                  default = 16;
+                  description = "Max outbound connections";
+                };
+              };
+              fetchConcurrency = mkOption {
+                type = types.ints.positive;
+                default = 1;
+                description = "Maximum number of concurrent fetches per peer connection";
+              };
+              gossipMaxAge = mkOption {
+                type = types.ints.positive;
+                default = 2 * 7 * 24 * 60 * 60; # 2 weeks
+                defaultText = "2 * 7 * 24 * 60 * 60";
+                description = "How long to keep a gossip message entry before pruning it";
+              };
+              maxOpenFiles = mkOption {
+                type = types.ints.positive;
+                default = 4096;
+                description = "Maximum number of open files";
+              };
+              rate = {
+                inbound = {
+                  capacity = mkOption {
+                    type = types.ints.positive;
+                    default = 1024;
+                    description = "Inbound capacity limit for a single connection";
+                  };
+                  fillRate = mkOption {
+                    type = types.numbers.positive;
+                    default = 5.0;
+                    description = "Inbound fill rate for a single connection";
+                  };
+                };
+                outbound = {
+                  capacity = mkOption {
+                    type = types.ints.positive;
+                    default = 2048;
+                    description = "Outbound capacity limit for a single connection";
+                  };
+                  fillRate = mkOption {
+                    type = types.numbers.positive;
+                    default = 10.0;
+                    description = "Outbound fill rate for a single connection";
+                  };
+                };
+              };
+              routingMaxAge = mkOption {
+                type = types.ints.positive;
+                default = 7 * 24 * 60 * 60; # 1 week
+                defaultText = "7 * 24 * 60 * 60";
+                description = "How long to keep a routing table entry before being pruned";
```

Comment:
```suggestion
                description = "How many seconds to keep a routing table entry before being pruned.";
```

Reply:
I copied verbatim the description from https://app.radicle.xyz/nodes/seed.radicle.garden/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5/tree/radicle/src/node/config.rs#L99 
Let's make an exception for clarity ;-)